graphql_compiler/tests/test_data_tools/schema.sql

```diff
@@ -44,8 +48,8 @@ CREATE PROPERTY Animal.net_worth Decimal
 CREATE INDEX Animal.net_worth NOTUNIQUE
```

Comment:
If `net_worth` is supposed to be `Decimal`, then create it as `Decimal` in the generated SQL commands as well. Right now, the data is initialized as floats, which introduces some potential of spurious and frustrating errors in the tests. See the GraphQL handling of Decimal inputs for an example of the OrientDB syntax to use.